### PR TITLE
ref(feedback): take platform from issue project

### PR DESCRIPTION
**Pull request by michellewzhang**
prevents this bug from happening (showing replay setup for a react native project)

SCR-20240507-ndnd

### CODE REVIEW

File: static/app/components/feedback/useFetchFeedbackData.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import {useEffect, useMemo} from 'react';
 
-import hydrateEventTags from 'sentry/components/feedback/hydrateEventTags';
+import hydrateFeedbackTags from 'sentry/components/feedback/hydrateFeedbackTags';
```
Comment (bruno-garcia) [inline static/app/components/feedback/useFetchFeedbackData.tsx:3]: Seems like an unrelated change? was this intentional?

**Review by bruno-garcia — APPROVED** [review]
LGTM other than 1 file seems unrelated

File: static/app/components/feedback/useFetchFeedbackData.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/components/feedback/useFetchFeedbackData.tsx:3]: renamed the file! bc it's not only event tags
